lib/neptune/dataset/dataset.py

```diff
@@ -21,9 +21,12 @@ def _load_dataset(dataset_url, format, **kwargs):
         LOG.warning(f'dataset_url is None, please check the url.')
         return None
     if format == 'txt':
-        LOG.info("dataset format is txt, now loading txt from "
-                 f"[{dataset_url}]")
-        return _load_txt_dataset(dataset_url)
+        LOG.info(
+            f"dataset format is txt, now loading txt from [{dataset_url}]")
+        if kwargs.get('with_image'):
```

Comment:
what does the parameter `with_image` mean?

Reply:
There already been a function _load_txt_dataset,  which will return the image rgb value instead of image path. In order to distinguish the two function, the parameter is added.

Reply:
then _load_txt_dataset => _load_txt_dataset_with_path?

Reply:
good suggestion.